_commands/tools/time.md

```diff
@@ -0,0 +1,123 @@
+---
+---
+
+time
+---
+`time` prints out the total time elapsed, the time used by system overhead and the time used to execute the utility.
+	
+### General Form:
+
+~~~ bash
+$ /usr/bin/time [ -apqvV ] [ -f FORMAT ] [ -o FILE ] [ --append ] [ --verbose ] [ --quiet ] [ --portability ]
+[ --format=FORMAT ] [ --output=FILE ] [ --version ] [ --help ] COMMAND [ ARGS ] 
+~~~
+
+<!--more-->
+
+### Situation
+
+If you simply want to output the time it takes to run a command
+	
+~~~ bash
+$ /usr/bin/time ./project -m < input.txt
+~~~
+
+### Meaning:
+'/usr/bin/time' calls the bash time command, './project' is the program you are trying to run,
+'-m' is a flag for that program and '< input.txt' is the input file for the program.
+Anything that appears after './project' is taken like it would be without using the time command.
+This will output the following:
+
+0.00user 0.00system 0:00.00elapsed 0%CPU (0avgtext+0avgdata 3312maxresident)k
```

Comment:
Either block terminal output

    like so

or at least make it not all appear on one ugly line

<img width="802" alt="screen shot 2016-03-25 at 10 33 17 am" src="https://cloud.githubusercontent.com/assets/5882053/14046037/0a4f0d22-f275-11e5-8551-64429c58d13d.png">